### PR TITLE
commited API to get timesheet by reporting manager id

### PR DESCRIPTION
**Pull request by devi-march**

### CODE REVIEW

File: backend/server/getemptimeesheet-manger-id.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const app = express();
+const port = 3000;
```
Comment (skStavir) [inline backend/server/getemptimeesheet-manger-id.js:3]: We need all Apis to be part of the running server. So either 1. add it to the existing file or 2. split the Apis to multiple files with common server and db files.
You can do 1 first and get it merged
Then pick up 2 based on your time availability.

**Review by skStavir — CHANGES_REQUESTED** [review]
Please make sure that all the Apis work when you start the server

File: backend/server/getemptimeesheet-manger-id.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+const express = require("express");
```
Comment (skStavir) [inline backend/server/getemptimeesheet-manger-id.js:1]: why do we still need this file?
Shouldn't it be removed?

**Comment by devi-march** [comment]
revertng

File: backend/server/getemptimeesheet-manger-id.js (hunk 2 above)
Comment (devi-march) [reply backend/server/getemptimeesheet-manger-id.js]: removed

**Review by skStavir — APPROVED** [review]

**Pull request merged**